packages/neos-ui-views/src/NodeInfoView/index.js

```diff
@@ -35,13 +35,34 @@ export default class NodeInfoView extends PureComponent {
             name: $get('properties._name', node) ? $get('properties._name', node) : '/'
         };
 
+        const nodeType = $get('nodeType', node);
```

Comment:
Better add this variable into properties object above, to make it more consistent